bootstrap/index.js

```diff
@@ -17,20 +17,32 @@ const withStatusCode = (execute) => (data, callback) => {
 const cacheOptions = envOptions()
 if (cacheOptions.enabled) logger.info('Cache enabled: ', cacheOptions)
 
+const withMiddleware = async (execute) => {
+  return await withCache(withStatusCode(execute))
+}
+
 // Execution helper async => sync
-const _executeSync = (execute) => {
-  // Add middleware
-  const _execute = withCache(withStatusCode(execute))
+const executeSync = (execute) => {
+  let _execute = undefined
   // Return sync function
-  return (data, callback) =>
-    _execute(data).then((result) => callback(result.statusCode, result.data))
+  return (data, callback) => {
+    // Init middleware only once
+    const init = _execute
```

Comment:
Won't this result in execute never having middleware wrapped around it?

Reply:
Init will check if `_execute` exists in the closure. If it does, it means it was already wrapped with middleware and it uses it, if not it wraps it with middleware.